docs/sphinx_doc/zh_CN/source/tutorial/204-service.md

```diff
@@ -46,6 +46,10 @@
 |                             | `openai_image_to_text` | 使用 OpenAI API 根据图片生成文字。
 |                             | `openai_text_to_audio` | 使用 OpenAI API 根据文本生成音频。
 |                             | `openai_audio_to_text` | 使用OpenAI API将音频转换为文本。
+| 推理                         | query_wolfram_alpha_short_answers  | 查询 Wolfram Alpha Short Answers API。                |
```

Comment:
Plz add more details about this API. For example, we cannot know the difference between short answer and simple APIs in this doc.

Reply:
Done.